crashlytics/README.md

```diff
@@ -1,22 +1,41 @@
 Firebase Crashlytics Quickstart
 =============================
```

Comment:
Please limit line lengths to 100 characters so the README.md is readable in any editor.